src/Vts/MonteCarlo/Sources/SurfaceEmittingBulkSources/Spherical/LambertianSurfaceEmittingSphericalSource.cs

```diff
@@ -103,9 +135,29 @@ public LambertianSurfaceEmittingSphericalSource(
                 translationFromOrigin,
                 initialTissueRegionIndex)
         {
+            _lambertOrder = lambertOrder;
             if (translationFromOrigin == null)
-            {
-            }
-        }        
-    }
+                SourceDefaults.DefaultPosition.Clone();

```

Comment:
@hayakawa16 Is the indenting messed up here or does it look that way due to the deleted code?

Reply:
I think it needs parentheses to follow our style rules.

Reply:
It looks okay in the source code.  However it is an "if" that has only one statement and that statement is on the next line.  I think we decided to move those to be on the same line as the "if".  I will do that.

Reply:
Yes! Same line will work also.